src/getHumanAge.test.js

```diff
@@ -7,4 +7,54 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be return current array [0, 0]  for input data(0, 0)', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should be return current array [0, 0]  for input data (14, 14)', () => {
```

Comment:
Replace 'should be return current array [0, 0]' with 'should return correct array [0, 0]' in the test description